Source/NSGlyphGenerator.m

```diff
@@ -93,13 +93,15 @@ - (void) generateGlyphsForGlyphStorage: (id <NSGlyphStorage>)storage
                         characterIndex: (NSUInteger*)index
 {
   // Try to get enough space for all glyphs
-  NSGlyph glyphs[2 * num];
+  // TESTPLANT-MAL-09132018: Would be nice to optimize this...
+  //     but we've been seeing stack overwrite crashes due to this...
+  NSGlyph *glyphs = NSZoneMalloc(NSDefaultMallocZone(), 2 * num * sizeof(NSGlyph));
```

Comment:
This is the same workaround I complained about up there ^, that if you're smashing the stack the solution is not to smash the heap instead.